record/gedcom-evidence/record-_RDOC.yaml

```diff
@@ -0,0 +1,27 @@
+%YAML 1.2
+---
+contact: https://github.com/glamberson/gedcom-evidence
+extension tags: _RDOC
+label: Research Documentation Record
+lang: en-US
+payload: null
+specification:
+- Research Documentation Record
+- 'A record documenting research reasoning, analysis, and decisions. _RDOC
+  records capture the "why" behind associations between evidence and
+  individuals, including proof arguments and conflict resolution.
+  This supports the Genealogical Proof Standard by documenting reasonably
+  exhaustive searches, complete source citations, analysis and correlation,
+  resolution of conflicts, and soundly reasoned conclusions.'
+substructures:
+  https://gedcom.io/terms/v7/CHAN: '{0:1}'
+  https://gedcom.io/terms/v7/DATE: '{0:1}'
+  https://gedcom.io/terms/v7/EXID: '{0:M}'
+  https://gedcom.io/terms/v7/NOTE: '{0:M}'
+  https://gedcom.io/terms/v7/SNOTE: '{0:M}'
+  https://gedcom.io/terms/v7/UID: '{0:M}'
+  https://github.com/glamberson/gedcom-evidence/_RACT: '{0:M}'
+superstructures: []
```

Comment:
_CONC and _ID and _RACT all list _RDOC as a substructure, so should this list them as superstructures?

Or is the intent to have those structures have substructures that point to an _RDOC? If so, that needs two URIs: one for the record (this one) and another for the pointer structure (compare pointer https://gedcom.io/terms/v7/SNOTE and record https://gedcom.io/terms/v7/record-SNOTE)